### ISSUE TITLE
AZW3 not downloadable in OPDS

### ISSUE THREAD

**Issue by RandomDennisLee** [janeczku/calibre-web#2115]
**Describe the bug/problem**
When viewing the OPDS feed from ebook readers (tested: moon reader, fb reader. Issue exists on both.), AZW3-formatted books are not available to download. The books are valid, and can be downloaded and opened via the web UI. epub and mobi work as expected.

**To Reproduce**
1. Load up OPDS feed in ebook reader.
2. Book is not visible, or metadata can be viewed but book is not downloadable.
3. Book can be downloaded and opened via web UI

**Logfile**
n/a

**Expected behavior**
Books are available and downloadable

**Screenshots**
n/a

**Environment (please complete the following information):**
 - OS: Windows 10
 - Python version: 3.9.6
 - Calibre-Web version: 0.6.12
 - Docker container: None
 - Special Hardware: n/a
 - Browser: moon reader, fb reader (android)

**Additional context**
n/a

**Comment by RandomDennisLee** [comment]
Update: It's a mess. Calibre web is serving up mime type "application/x-mobi8-ebook", which is deprecated and replaced by "application/vnd.amazon.mobi8-ebook".

Moon and fb reader are both expecting mime type "application/azw3", which is definitely not right. 

Librera reader supports "application/x-mobi8-ebook" and "application/azw3", but not yet "application/vnd.amazon.mobi8-ebook", which should be the technically correct one.

**Comment by OzzieIsaacs** [comment]
Calibre itself also seems to use application/x-mobi8-ebook
https://github.com/kovidgoyal/calibre/blob/ba89a2b970aedc507b38fdc4a80b03869a26a355/resources/calibre-mimetypes.xml#L37

**Comment by RandomDennisLee** [comment]
Yeah, OPDS support is extremely hit and miss across most of the software. Librera reader has just added support for "application/vnd.amazon.mobi8-ebook" yesterday, based on an issue I opened there at the same time I did this one.

You can check for yourself at https://www.iana.org/assignments/media-types/media-types.xhtml that "application/x-mobi8-ebook" was deprecated in 2016.

`Calibre itself also seems to use application/x-mobi8-ebook`

I forgot to check / submit an issue for calibre as I don't use their Web interface.

**Comment by OzzieIsaacs** [comment]
> Yeah, OPDS support is extremely hit and miss across most of the software.

I normally try to be inline with Calibre itself, as it's also some sort of standard. So if they update the mimetype I will also do it.
